### ISSUE TITLE
Display for PAS photodiode signal

### ISSUE THREAD

**Issue by nicholaslwagner** [NOAA-CSD/nicholas#19]
The UI needs a display of the Photo diode signal for each PAS channels and time series of the LRMS signal. We also need to check that these signal are appearing on the routing board and look good on a scope.

**Comment by nicholaslwagner** [comment]
I check this signal on the routing board and they were very small. (with the laser gain on the interface boards set to max).  I removed OD2 filters from in front of the PDs. I adjusted the laser gain such that the LRMS_chan1 = 0.5 v and the LRMS_chan2 = 0.8 volt (chan2 could not be adjusted lower than 0.7 volts).

The next step is to check that the LRMS value is read correctly by the DAQ and UI.
